packages/desktop-client/src/components/filters/NameFilter.tsx

```diff
@@ -0,0 +1,80 @@
+// @ts-strict-ignore
+import React, { useRef, useEffect } from 'react';
+
+import { theme } from '../../style';
+import { Button } from '../common/Button';
+import { Input } from '../common/Input';
+import { MenuTooltip } from '../common/MenuTooltip';
+import { Stack } from '../common/Stack';
+import { Text } from '../common/Text';
+import { FormField, FormLabel } from '../forms';
+
+export function NameFilter({
+  onClose,
+  menuItem,
+  name,
+  setName,
+  adding,
+  onAddUpdate,
+  err,
+}: {
+  onClose: () => void;
+  menuItem: string;
+  name: string;
+  setName: (item: string) => void;
+  adding: boolean;
+  onAddUpdate: () => void;
+  err: string | null;
+}) {
+  const inputRef = useRef<HTMLInputElement>();
```

Comment:
```suggestion
  const inputRef = useRef<HTMLInputElement>(null);
```

If we do this, I think this fixes the strict error on this file, and we can drop the first line ignore